virtual_rainforest/main.py

```diff
@@ -333,6 +244,13 @@ def vr_run(
     # Container to store paths to continuous data files
     continuous_data_files = []
 
+    # Only variables in the data object that are updated by a model should be output
+    all_variables = [
+        models_cfd[model_nm].vars_updated for model_nm in models_cfd.keys()
+    ]
```

Comment:
It would be better to just leave `all_variables` as a generator expression so that you don't have to construct an intermediate `list`:

```suggestion
    all_variables = (
        models_cfd[model_nm].vars_updated for model_nm in models_cfd.keys()
    )
```